README.md

```diff
@@ -1,185 +1,192 @@
+SELECT time_bucket('1 day', time) AS bucket,
+  avg(temperature) AS avg_temp
+FROM weather_conditions
+GROUP BY bucket
+ORDER BY bucket ASC;
 ```
 
-In addition, TimescaleDB includes additional functions for time-series
-analysis that are not present in vanilla PostgreSQL. (For example, the `time_bucket` function above.)
+See more:
 
-- [Quick start: Basic operations](https://tsdb.co/GitHubTimescaleBasicOperations)
-- [Reference examples](https://tsdb.co/GitHubTimescaleWriteData)
-- [TimescaleDB API](https://tsdb.co/GitHubTimescaleAPI)
+- [About time buckets](https://docs.timescale.com/use-timescale/latest/time-buckets/about-time-buckets/)
+- [API reference](https://docs.timescale.com/api/latest/hyperfunctions/time_bucket/)
+- [All TimescaleDB features](https://docs.timescale.com/use-timescale/latest/)
+- [Tutorials](https://docs.timescale.com/tutorials/latest/)
 
-### Installation
+# Create continuous aggregates
 
-[Timescale Cloud](https://tsdb.co/GitHubTimescale), a fully-managed TimescaleDB in the cloud, is
-available via a free trial. Create a PostgreSQL database in the cloud with TimescaleDB pre-installed
-so you can power your application with TimescaleDB without the management overhead.
+Continuous aggregates are designed to make queries on very large datasets run faster. They use PostgreSQL [materialized views](https://www.postgresql.org/docs/current/rules-materializedviews.html) to continuously and incrementally refresh a query in the background, so that when you run the query, only the data that has changed needs to be computed, not the entire dataset.
 
-TimescaleDB is also available pre-packaged for several platforms such as Linux, Windows, MacOS, Docker, and
-Kubernetes. For more information, see [Install TimescaleDB](https://docs.timescale.com/self-hosted/latest/install/).
+See more:
 
-To build from source, see [Building from source](https://github.com/timescale/timescaledb/blob/main/docs/BuildSource.md).
+- [About continuous aggregates](https://docs.timescale.com/use-timescale/latest/continuous-aggregates/)
+- [API reference](https://docs.timescale.com/api/latest/continuous-aggregates/create_materialized_view/)
 
-## Resources
+# Tier data
 
-### Architecture documents
+TimescaleDB tiered storage architecture includes a standard high-performance storage tier and a low-cost object storage tier. You can migrate rarely used data to the object storage to cut costs. Data is tiered on the level of chunks, that is, individual parts of tables. This means that a single table can be spread among storage tiers for ultimate cost optimization.
 
-- [Basic TimescaleDB Features](tsl/README.md)
-- [Advanced TimescaleDB Features](tsl/README.md)
-- [Testing TimescaleDB](test/README.md)
+<img src="https://assets.timescale.com/docs/images/tiered-storage.png" alt="tiered-storage"/>
+
+See more:
+
+- [About tiered storage](https://docs.timescale.com/use-timescale/latest/data-tiering/)
 
-### Useful tools
+# Ensure high availability
```

Comment:
IMHO this section should not be added since it is a cloud feature not an extension feature.

Reply:
++ to Fabrizio's comment. HA is not a feature shipped with the extension, we can remove it.